developer/src/kmc/src/util/NodeCompilerCallbacks.ts

```diff
@@ -121,6 +122,14 @@ export class NodeCompilerCallbacks implements CompilerCallbacks {
 
     this.messages.push({...event});
 
+    // report fatal errors to Sentry, but don't display; note, it won't be
+    // reported if user has disabled the Sentry setting
+    if(CompilerError.severity(event.code) == CompilerErrorSeverity.Fatal) {
+      // this is async so returns a Promise, we'll let it resolve
+      // in its own time, and it can print its message then
+      KeymanSentry.reportException(event.exceptionVar, false);
+    }
```

Comment:
+1… and the unit test callbacks don't go through this path, so they won't trigger.